OpenRA.Renderer.Sdl2/ErrorHandler.cs

```diff
@@ -56,8 +56,10 @@ public static void WriteGraphicsLog(string message)
 			Log.Write("graphics",  "Vendor: {0}", GL.GetString(StringName.Vendor));
 			if (GL.GetString(StringName.Vendor).Contains("Microsoft"))
 			{
-				Log.Write("graphics", "Note:  The default driver provided by Microsoft does not include full OpenGL support.\n"
-					+ "Please install the latest drivers from your graphics card manufacturer's website.\n");
+				String msg = "";
```

Comment:
You should use `var` according to https://github.com/OpenRA/OpenRA/wiki/Coding-Standard. As this does not get re-used anywhere you may just want to skip storing this into a local variable.